### DISCUSSION TITLE
Debugging a Jupyter Notebook with Python 3.6?

### DISCUSSION THREAD

**Discussion by brunomorampc** [microsoft/vscode-jupyter#14331] [Questions and Answers]
My project must use Python 3.6, which means I can't install ipykernel=6.0.0 which is necessary to debug the cells (see here https://github.com/microsoft/vscode-jupyter/wiki/Setting-Up-Run-by-Line-and-Debugging-for-Notebooks) is there a workaround to that?

Thanks

**Comment by DonJayamanne** [comment]
I think your option would be to install an old version of Juptyer Extension, Pytohn Extension and VS Code
